.github/workflows/go.yml

```diff
@@ -22,6 +22,14 @@ jobs:
     - name: Check out code into the Go module directory
```

Comment:
what's the diff between this and go-scheduled.yml?

Reply:
Scheduled tests run once a day. This runs on PRs and code pushes to master branch.

Reply:
Scheduled tests also run on extended dataset.